aerleon/lib/openconfig.py

```diff
@@ -70,12 +70,14 @@ def RecursiveDict() -> DefaultDict[Any, Any]:
 )
 aclEntries = TypedDict("aclEntries", {"acl-entry": List[ACLEntry]})
 ACLSetConfig = TypedDict("ACLSetConfig", {"name": str, "type": str})
-ACLSet = TypedDict(
-    "ACLSet", {"acl-entries": aclEntries, "config": ACLSetConfig, "name": str, "type": str}
-)
 
 
-class Term(aclgenerator.Term):
+# ACLSet = TypedDict(
```

Comment:
Why the comment out of ACLSet here?

Reply:
Because it's no longer being used - with the rewrite, the dict can contain either openconfig or srlinux ACL entries (different structure)

I'm not sure if/how validation works, I didn't find a difference in having these specs be correct or not - may need further review

Reply:
The specs are used for type hinting. https://peps.python.org/pep-0484/
Eventually once we have things fully type hinted we can use mypy or some other tool for static type analysis which will give us more confidence in code correctness.

Reply:
You will want to try and get them things right for the different modules. You should be able to overwrite the ACLSet used in nokiasrl with a Type of the correct structure.